### PR TITLE
slide46: Modify drag and drop

### PR DESCRIPTION
**Pull request by Sanjeev2487**
Make changes to slide 46 drag and drop

### CODE REVIEW

File: slide46/index.html [hunk 1]
```diff
@@ -1,87 +1,60 @@
...
+        <div id = "#11" class = "dropBox" style = "top: 41%;left: 69%;width: 17%; height: 4%">#11</div>
+      </div>
+      <div class= "answer-container">
+        <div class="word-bank">
+          <div class="answer" id="4">Vacuole</div>
+          <div class="answer" id="3">Chloroplast</div>
+          <div class="answer" id="2">Cell membrane</div>
+          <div class="answer" id="1">Golgi apparatus</div>
+          <div class="answer" id="11">Mitochondria</div>
+          <div class="answer" id="10">Nucleus</div>
```
Comment (dsandeep97) [inline slide46/index.html:45]: Should be Cytosol

File: slide46/index.html [hunk 2]
```diff
@@ -1,87 +1,60 @@
...
-      <div class="answer" id="7" >Nucleus</div>
-      <div class="answer" id="6" >Rough ER</div>
+    <!-- Include touch punch -->
+    <script type="text/javascript" src="js/jquery.ui.touch-punch.js"></script>
+    <script type="text/javascript" src="js/jquery.ui.touch-punch.min.js"></script>
+
+    <!-- drag and drop animation script -->
+    <script type="text/javascript" src="js/slide46.js"></script>
+
+    <div id="master-container">
```
Comment (dsandeep97) [inline slide46/index.html:24]: we have only an image which doesn't have any heading/text. Do you think we should add some heading?

**Review by dsandeep97 — CHANGES_REQUESTED** [review]
Also, lets remove the additional file DD_slide46.js

File: slide46/index.html (hunk 2 above)
Comment (Sanjeev2487) [reply slide46/index.html:24]: Yes. I should add a heading. Any suggestions based on the image?

Comment (dsandeep97) [reply slide46/index.html:24]: The other one has "Eukaryotic Cell Review". I don't know what this is?  (Animal Cell?) May be Animal Cell Review could be heading?

**Pull request merged**